core/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -160,21 +160,6 @@
    * <p>If this method is not called, the builder will use {@link PickFirstBalancerFactory}
    * for the channel.
    *
-   * <p>This overrides what was set by {@link #loadBalancerFactory(LoadBalancer2.Factory)}.
-   *
-   * <p>Calling this will revert the channel back to the original (LBv1) implementation.
-   */
-  @ExperimentalApi("https://github.com/grpc/grpc-java/issues/1771")
-  public abstract T loadBalancerFactory(LoadBalancer.Factory loadBalancerFactory);
```

Comment:
this has to remain abstract, since changedetector tests internally will complain about it.